example/fittemplate.md

```diff
@@ -0,0 +1,250 @@
+---
+title: How to create a head model if you do not have an individual MRI
+tags: [example, source]
+---
+
+# How to create a head model if you do not have an individual MRI
+
+## Introduction
+
+A volume conduction model of the head is required for source reconstruction. Ideally you base the
+headmodel on the individual's anatomical MRI, but that is not always available. In the case of EEG you can use a template head model and fit your measured electrodes (or template electrodes) on the scalp of the template model. For MEG you cannot simply use a template head model, since the distance between the head and the (fixed) MEG sensors depends on the head size.
+
+In this example we will show two ways on how to create an individual head model on the basis of surface
+data acquired with the Polhemus. Both methods determine the translation, rotation and scaling of the
+template head model to fit the Polhemus head shape, but the second method applies a more detailed spatial scaling.
+
+## Download
+
+Both approaches will be demonstrated using the same dataset, which you can find on the
+[ftp server](ftp://ftp.fieldtriptoolbox.org/pub/fieldtrip/tutorial/epilepsy). More
+information on this dataset can be found [here](/tutorial/epilepsy/).
+
+## Loading the data
+
+Before starting with FieldTrip, it is important that you set up your
+[MATLAB path](/faq/should_i_add_fieldtrip_with_all_subdirectories_to_my_matlab_path) properly.
+
+You can load the head shape measured during the MEG recording with the Polhemus and a
+template volume conduction model. We have to ensure that they have consistent units, hence
+we will convert the units into mm.
+
+    polhemus = ft_read_headshape('epilepsy/case1/ctf_data/case1.pos');
+    polhemus = ft_convert_units(polhemus,'mm');
+
+    template = ft_read_vol('standard_bem.mat');
+    template = ft_convert_units(template,'mm');
+
+## Coregistration
```

Comment:
you could also take the template immediately following coregistration. In that case translation and rotation have been done, but no scaling.